velox/common/file/File.cpp

```diff
@@ -68,9 +68,7 @@ void ReadFile::preadv(
   for (size_t i = 0; i < regions.size(); ++i) {
     const auto& region = regions[i];
     auto& output = iobufs[i];
-    output = folly::IOBuf(folly::IOBuf::CREATE, region.length);
```

Comment:
@Yohahaha `ReadFile::preadv` is expected to create`IOBuf`.  This is not beneficial in `LocalReadFile`, but the disaggregate storages will benefit from this, because they keep their own `IOBuf` around.
CC: @DanielMunozT